### DISCUSSION TITLE
Is it possible to log to /var/log/syslog?

### DISCUSSION THREAD

**Discussion by adampatterson** [spatie/laravel-backup#1534] [Q&A] [answered]
I increased the frequency of backups on a staging server to test before moving it to production.

The server was being backed up at daily at `00:00` and I have increased that to hourly.

```
$schedule->command('backup:clean')->hourly()->at(30);
$schedule->command('backup:run')->hourly();
```

Looking in S3, my backups were showing up maybe 3 min after `00:00`, but now that I made this change today's backup was not created. I ran `backup:run` manually and it ran just fine.

The task `* * * * * /usr/bin/php ....`

I then checked the `/var/log/syslog` and it's recording the base schedule cron every min `... schedule:run >> /dev/null 2>&1`

Is there a logging option that I can configure for the backups independently from the site?

**Comment by erikn69** [comment, marked as answer]
Use `appendOutputTo()`
```php
$schedule->command('backup:clean')->hourly()->appendOutputTo(base_path('storage/backup/log.txt'));
$schedule->command('backup:run')->hourly()->appendOutputTo(base_path('storage/backup/log.txt'));
```
Or a custom notifications to logger

**Comment by adampatterson** [comment]
Thanks, @erikn69 I actually wrote a wrapper command that worked as expected. Thanks, for the suggestions!

I was also having issues because `->hourly()->at(30)` is invalid it should have been `->hourly(30)`
